connectors/http/http-base/src/main/java/io/camunda/connector/http/base/blocklist/DefaultHttpBlocklistManager.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.http.base.blocklist;
+
+import com.google.api.client.http.GenericUrl;
+import io.camunda.connector.http.base.blocklist.block.Block;
+import io.camunda.connector.http.base.blocklist.factory.BlockFactory;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class DefaultHttpBlocklistManager implements HttpBlockListManager {
+
+  private static final String BLOCK_PREFIX = "CONNECTORS_HTTP_BLOCK_";
```

Comment:
Lets use `CAMUNDA_CONNECTOR_HTTP_` as the prefix.

Reply:
Ah, sorry misunderstanding. Yes, I meant: `CAMUNDA_CONNECTOR_HTTP_BLOCK_URL_NAME`